.circleci/config.yml

```diff
@@ -339,6 +339,25 @@ jobs:
           tag: latest
       - run: heroku container:release -a simple-hub-staging simple-hub
 
+  e2e-test-w3t-production:
```

Comment:
`staging`? cc @kerzhner

Reply:
I think out of scope for this PR, which follows conventions that exist on the base branch. If we were to go with 'staging' we should also make changes in quite a few other places (best done in a separate PR).